### PR TITLE
Move more string consts into cata_string_consts.h

### PR DESCRIPTION
**Pull request by Ramza13**
#### Summary
SUMMARY: Infrastructure "Continue moving string consts into cata_string_consts.h"

#### Purpose of change
Centralize string consts in one location for good coding practice.

#### Describe the solution
Migrated a ton of string consts into cata_string_consts.h

### CODE REVIEW

File: src/character.cpp [hunk 1]
```diff
@@ -4167,11 +4168,12 @@ void Character::check_needs_extremes()
             g->events().send<event_type::dies_of_thirst>( getID() );
             hp_cur[hp_torso] = 0;
         } else if( get_thirst() >= 1000 && calendar::once_every( 30_minutes ) ) {
-            add_msg_if_player( m_warning, _( "Even your eyes feel dry…" ) );
+            add_msg_if_player( m_warning, _( "Even your eyes feel dry�" ) );
+            add_msg_if_player( m_warning, _( "Even your eyes feel dry�" ) );
```
Comment (ifreund) [inline src/character.cpp]:
```suggestion
```

File: src/character.cpp [hunk 2]
```diff
@@ -4167,11 +4168,12 @@ void Character::check_needs_extremes()
             g->events().send<event_type::dies_of_thirst>( getID() );
             hp_cur[hp_torso] = 0;
         } else if( get_thirst() >= 1000 && calendar::once_every( 30_minutes ) ) {
-            add_msg_if_player( m_warning, _( "Even your eyes feel dry…" ) );
+            add_msg_if_player( m_warning, _( "Even your eyes feel dry�" ) );
```
Comment (ifreund) [inline src/character.cpp]:
```suggestion
            add_msg_if_player( m_warning, _( "Even your eyes feel dry…" ) );
```

File: src/character.cpp [hunk 3]
```diff
@@ -4167,11 +4168,12 @@ void Character::check_needs_extremes()
...
+            add_msg_if_player( m_warning, _( "Even your eyes feel dry�" ) );
         } else if( get_thirst() >= 800 && calendar::once_every( 30_minutes ) ) {
             add_msg_if_player( m_warning, _( "You are THIRSTY!" ) );
         } else if( calendar::once_every( 30_minutes ) ) {
-            add_msg_if_player( m_warning, _( "Your mouth feels so dry…" ) );
+            add_msg_if_player( m_warning, _( "Your mouth feels so dry�" ) );
```
Comment (ifreund) [inline src/character.cpp]:
```suggestion
            add_msg_if_player( m_warning, _( "Your mouth feels so dry…" ) );
```

File: src/character.cpp [hunk 4]
```diff
@@ -4183,7 +4184,7 @@ void Character::check_needs_extremes()
             mod_fatigue( -10 );
             fall_asleep();
         } else if( get_fatigue() >= 800 && calendar::once_every( 30_minutes ) ) {
-            add_msg_if_player( m_warning, _( "Anywhere would be a good place to sleep…" ) );
+            add_msg_if_player( m_warning, _( "Anywhere would be a good place to sleep" ) );
```
Comment (ifreund) [inline src/character.cpp]:
```suggestion
            add_msg_if_player( m_warning, _( "Anywhere would be a good place to sleep…" ) );
```

Comment (Ramza13) [reply src/character.cpp]: Thank you very much!

**Comment by ZhilkinSerg** [comment]
Something is wrong here - multiple tests are failing now.

**Comment by Ramza13** [comment]
This is way too big and possibly causing test failures so I am going to break it up into smaller chunks that can easily be reviewed/merged.

**Pull request closed without merging**